PROTO_tests/tests/sort_test.py

```diff
@@ -0,0 +1,71 @@
+import numpy as np
+import pytest
+
+import arkouda as ak
+from arkouda.sorting import SortingAlgorithm
+
+
+class TestSort:
+    @pytest.mark.parametrize("size", pytest.prob_size)
+    def test_sort_dtype(self, size):
+        for dtype in ak.int64, ak.uint64:
+            pda = ak.randint(0, 100, size, dtype)
+            for algo in SortingAlgorithm:
+                spda = ak.sort(pda, algo)
+                assert ak.is_sorted(spda)
+
+        pda_float = ak.array(np.random.rand(100) * 10000)
+        for algo in SortingAlgorithm:
+            spda = ak.sort(pda_float, algo)
+            assert ak.is_sorted(spda)
+
+        pda = ak.randint(0, 100, size, dtype=ak.uint64)
+        pda_bigint = pda + 2**200
+        for algo in SortingAlgorithm:
+            sorted_pda = ak.sort(pda, algo)
+            sorted_bi = ak.sort(pda_bigint, algo)
+            assert (sorted_bi - 2**200).to_list() == sorted_pda.to_list()
```

Comment:
I don't think it's super important to parameterize on `size` (especially if we're only drawing from `0-100` no matter what), but I think parameterizing on `algo` and `dtype` would make this way more compressed

Every function in this test should be able to parameterized on `algo`. If you want an example of how to parameterize on `algo` and `dtype`, I already converted `coargsort` and I think it does something similar

Reply:
Updated